ddprof-lib/src/main/cpp/wallClock.h

```diff
@@ -165,6 +165,7 @@ class WallClockJVMTI : public BaseWallClock {
     struct ThreadEntry {
         ddprof::VMThread* native;
         jthread java;
+        int tid;
```

Comment:
Just a nit - maybe premature optimization but can we 'unionize' the `native` and `tid` since they will never be used at the same time?